src/table/builder.rs

```diff
@@ -22,7 +22,7 @@ impl Header {
         bytes.put_u32_le((self.overlap as u32) << 16 | self.diff as u32);
     }
 
-    pub fn decode(&mut self, bytes: &mut Bytes) {
+    pub fn decode(&mut self, mut bytes: &[u8]) {
```

Comment:
Why change?

Reply:
It seems that we could retain original design once we changed the way of constructing key in previous commits. I've changed that back.